src/routers/auth.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+
+const router = express.Router();
+
+const authController = require('../controllers/auth');
+
+// POST route for /signin
```

Comment:
In your comments on all router files, can you mention the complete path?
So for example, this comment should say: `// POST route for /api/auth/signin`

Then the comment becomes more helpful in addition to the code below it.